src/test/java/de/orchound/gameoflife/game/BoardTest.java

```diff
@@ -1,42 +1,90 @@
-package de.orchound.gameoflife.game;
-
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
-
-import static org.junit.jupiter.api.Assertions.*;
-
-class BoardTest {
-
-	Board board = new Board(5, 5);
-
-	@BeforeEach
-	void setUp() {
-		for (int i = 0; i < board.target.length; i++) {
-			for (int j = 0; j < board.target[0].length; j++) {
-				board.target[i][j] = false;
-			}
-		}
-	}
-
-	@Test
-	void testSetCellAlive() {
-		board.setCellAlive(1, 0);
-		assertTrue(board.target[1][0]);
-	}
-
-	@Test
-	void testUpdate() {
-		board.setCellAlive(1, 2);
-		board.setCellAlive(2, 2);
-		board.setCellAlive(3, 2);
-
-		board.update();
-
-		assertTrue(board.target[2][1]);
-		assertTrue(board.target[2][2]);
-		assertTrue(board.target[2][3]);
-
-		assertFalse(board.target[1][2]);
-		assertFalse(board.target[3][2]);
-	}
-}
+package de.orchound.gameoflife.game;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class BoardTest {
+
+	Board board = new Board(5, 5);
+
+	@BeforeEach
+	void setUp() {
+		for (int i = 0; i < board.target.length; i++) {
+			for (int j = 0; j < board.target[0].length; j++) {
+				board.target[i][j] = false;
+			}
+		}
+	}
+
+	@Test
+	void testSetCellAlive() {
+		board.setCellAlive(1, 0);
+		assertTrue(board.target[1][0]);
+	}
+
+	@Test
+	void testUpdate() {
+		board.setCellAlive(1, 2);
+		board.setCellAlive(2, 2);
+		board.setCellAlive(3, 2);
+
+		board.update();
+
+		assertTrue(board.target[2][1]);
+		assertTrue(board.target[2][2]);
+		assertTrue(board.target[2][3]);
+
+		assertFalse(board.target[1][2]);
+		assertFalse(board.target[3][2]);
+	}
+
+
+	/**
+	 * Verifies that values on the right edge of the board are treated as neighbors to values on the left edge of the
+	 * board.
+	 */
+	@Test
+	void testHorizontalWrapping() {
+		board.setCellAlive(1, 0);
+		board.setCellAlive(2, 0);
+		board.setCellAlive(3, 0);
+
+		board.update();
+
+		assertTrue(board.target[2][4]);
+		assertTrue(board.target[2][0]);
+		assertTrue(board.target[2][1]);
+
+		assertFalse(board.target[1][0]);
+		assertFalse(board.target[3][0]);
+	}
+
+	/**
+	 * Verifies that values on the top edge of the board are treated as neighbors to values on the bottom edge of the
+	 * board.
+	 */
+	@Test
+	void testVerticalWrapping() {
+		Board board = new Board(20, 10);
+		for (int i = 0; i < board.target.length; i++) {
+			for (int j = 0; j < board.target[0].length; j++) {
+				board.target[i][j] = false;
+			}
+		}
+

```

Comment:
Simply move initialize the test board with `new Board(20, 10)`. All tests benefit when they are not only testing the special case of having a square board.
```suggestion
```